File: src/main/java/org/radarbase/appserver/dto/protocol/ReferenceTimestampType.java
package org.radarbase.appserver.dto.protocol;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum ReferenceTimestampType {
    @JsonProperty("date")
    DATE    ,
    @JsonProperty("datetime")
    DATETIME,
    @JsonProperty("datetimeutc")
    DATETIMEUTC,
    @JsonProperty("now")
    NOW,
    @JsonProperty("today")
    TODAY
}

File: appserver-jersey/src/main/kotlin/org/radarbase/appserver/jersey/dto/protocol/ReferenceTimestampType.kt
package org.radarbase.appserver.jersey.dto.protocol

import com.fasterxml.jackson.annotation.JsonProperty

@Suppress("unused")
enum class ReferenceTimestampType {
    @JsonProperty("date")
    DATE,

    @JsonProperty("datetime")
    DATETIME,

    @JsonProperty("datetimeutc")
    DATETIMEUTC,

    @JsonProperty("now")
    NOW,

    @JsonProperty("today")
    TODAY,
}
